### DISCUSSION TITLE
How to apply cheat to ISO ?

### DISCUSSION THREAD

**Discussion by ghost** [PS2-Widescreen/OPL-Widescreen-Cheats#41] [General]
Hello . Can we apply your cheats directly to the ISO and play on PS2 or PS3 ? Great job , by the way ! And a suggestion , maybe you can put a list of supported games in the readme . Thanks .

**Comment by israpps** [comment]
> Hello . Can we apply your cheats directly to the ISO and play on PS2 or PS3 ? Great job , by the way ! And a suggestion , maybe you can put a list of supported games in the readme . Thanks .

1) no, those cheats can't be applied to the game file directly. Not sure if there's a workaround to make that happen

2) there is already a [list of supported games](https://github.com/PS2-Widescreen/OPL-Widescreen-Cheats/blob/main/cheat_list.MD)

**Comment by firstreaper01** [comment]
I've downloaded the pack with games widescreen patches, but i don't know how to activate on emulator, can someone help me? cause i read the instructions, but I dont get how to do it on emulator pcsx2

**Reply by Mikaela** [reply]
Look up how to use cheats with PCSX2 as wide-screen patches count as cheats

**Reply by israpps** [reply]
> I've downloaded the pack with games widescreen patches, but i don't know how to activate on emulator, can someone help me? cause i read the instructions, but I dont get how to do it on emulator pcsx2

Use the cheats package bundled with PCSX2

This repository is intended for usage on real hardware via OPL or any other cheat capable game loader

**Reply by firstreaper01** [reply]
Thnaks, I manage to activate, I downloaded a version OPL and it's for Console, after this, I find a Pack for PCSX2 and now It's works fine. Tks.
